client/src/index.jsx

```diff
@@ -17,14 +20,25 @@ class GalleryView extends React.Component {
     });
   }
 
+  fetch(location) {
```

Comment:
This looks great! I don't have any noteworthy feedback except potentially implementing a catch function for the Axios GET request to console log errors!

Reply:
Thank you!